packages/quick_blue/windows/quick_blue_plugin.cpp

```diff
@@ -141,15 +141,19 @@ class QuickBluePlugin : public flutter::Plugin, public flutter::StreamHandler<En
   std::unique_ptr<flutter::BasicMessageChannel<EncodableValue>> message_connector_;
 
   std::unique_ptr<flutter::EventSink<EncodableValue>> scan_result_sink_;
+  std::unique_ptr<flutter::EventSink<EncodableValue>> availability_change_sink_;
 
   Radio bluetoothRadio{ nullptr };
+  RadioState oldBluetoothRadioState = RadioState::Unknown;
```

Comment:
Remove `oldBluetoothRadioState`

Reply:
@Sunbreak RadioState fired twice , whenever we turn on/off bluetooth , 
also mentioned [here](https://stackoverflow.com/a/67723902/13838294) , so that's why using oldBluetoothRadioState to prevent sending same event twice

Reply:
Then a `// FIXME` or comment would be nice